pkg/webhook/validate.go

```diff
@@ -143,9 +143,9 @@ func dynamicValidateSriovNetworkNodePolicy(cr *sriovnetworkv1.SriovNetworkNodePo
 	if !nodesSelected {
 		return false, fmt.Errorf("no matched node is selected by the nodeSelector in CR %s", cr.GetName())
 	}
-	if !interfaceSelected {
-		return false, fmt.Errorf("no matched NIC is selected by the nicSelector in CR %s", cr.GetName())
-	}
+	//if !interfaceSelected {
```

Comment:
We may need separate functions for validating or mutating new accelerator custom resource since the CRD is different from node policy. The comment-outs can be removed.

Reply:
you are right I miss this one sorry